### ISSUE TITLE
Bluetooth Audio disables Lightbulb for a moment

### ISSUE THREAD

**Issue by Dyrosis** [Tyrrrz/LightBulb#96]
When changing to/from a bluetooth audio device (and other random systems I haven't identified conclusively) lightbulb disables until it polls and re-enables itself 5 seconds later. The only trigger I've definitely found is when I connect or disconnect a bluetooth audio device lightbulb disables until (I assume) the gamma polling kicks in. There are other triggers where I get this issue, but this is the only one I've isolated.

**Comment by Tyrrrz** [comment]
Hi, yes, there are random things in the system that reset gamma. Lightbulb handles some of the more predictable ones (e.g. changing resolution or restoring computer from sleep), but for other cases the gamma polling ensures the gamma doesn't get reset for longer than 5 seconds, at most.

**Comment by Tyrrrz** [comment]
@Dyrosis if you're still around, can you check if this still happens with [this build](https://github.com/Tyrrrz/LightBulb/suites/298976768/artifacts/260352)?

**Comment by Dainius14** [comment]
Tested connecting to and disconnecting from my BT speaker, switching different audio devices on a single display and multiple display setups. Could not reproduce the issue and Lightbulb did not get disabled for a single second. To see if what exactly was the issue I tested the same routine (apart from multimonitor thing) with commit #df4eafb and also could not reproduce the issue. Soo..... Probably need OP to get some more input.
